### ISSUE TITLE
Windows 7, Visual Studio 2010, and "error C2065: uint32_t: undeclared identifier"

### ISSUE THREAD

**Issue by noloader** [weidai11/cryptopp#608]
Based on [cryptopp610 compilation issue in Visual Studio 2010](https://groups.google.com/d/msg/cryptopp-users/2adounqeh-E/5iN3ti8MBQAJ) from the mailing list. Windows 7, Visual Studio 2010:

```
> nmake /f cryptest.nmake
        cl.exe /nologo /W4 /wd4511 /wd4156 /D_MBCS /Zi /TP /GR /EHsc /DNDEBUG /D
_NDEBUG /Oi /Oy /O2 /MT /FI sdkddkver.h /c aria.cpp
...
aria.cpp(224) : error C2065: 'uint32_t' : undeclared identifier
aria.cpp(224) : error C2059: syntax error : ')'
NMAKE : fatal error U1077: '"c:\Program Files\Microsoft Visual Studio 10.0\VC\BI
N\cl.exe"' : return code '0x2'
Stop.
```

**Comment by noloader** [comment]
Cleared at [Commit 506f90bcc804](https://github.com/weidai11/cryptopp/commit/506f90bcc804).

**Comment by noloader** [comment]
The initial fix was provided at [Commit 506f90bcc804](https://github.com/weidai11/cryptopp/commit/506f90bcc804). The long term fix was to supply signed types, and it was cleared at [Commit 3958fad099f1](https://github.com/weidai11/cryptopp/commit/3958fad099f1).
